TodoApp/TodoApp.Api.Tests/Controllers/ItemListController.test.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using NUnit.Framework;
+using TodoApp.Api.Controllers;
+using TodoApp.Api.Models;
+using TodoApp.Api.Tests.Comparers;
+
+namespace TodoApp.Api.Tests.Controllers
+{
+    [TestFixture]
+    public class ItemListControllerTest
+    {
+        private ItemListController _controller;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _controller = new ItemListController
+            {
+                Request = new HttpRequestMessage(),
+                Configuration = new HttpConfiguration()
+            };
+        }
+
+        [Test]
+        public async Task GetAllItems_AllItemsReturned()
+        {
+            var expectedItems = new[]
+            {
+                new ItemModel {Id = Guid.Parse("c5cc89a0-ab8d-4328-9000-3da679ec02d3"), Text = "Make a cofee"},
+                new ItemModel {Id = Guid.Parse("55b0d56d-48d7-4f93-bd73-e4b801e26faa"), Text = "Make second coffee"},
+                new ItemModel {Id = Guid.Parse("886c2ea5-a639-4334-8c51-d3ee4e49acb9"), Text = "Make third cofffee"},
+                new ItemModel {Id = Guid.Parse("250be0cc-438e-46cc-a0fe-549f4d3409e2"), Text = "Coffee is awesome as well as Kentico is"}
+            };
+
+            var result = await _controller.GetAllItems();
+            var message = await result.ExecuteAsync(CancellationToken.None);
+            var actualStatusCode = message.StatusCode;
+            message.TryGetContentValue(out ItemModel[] actualItems);
+
+            Assert.That(actualStatusCode, Is.EqualTo(HttpStatusCode.OK));
+            Assert.That(actualItems, Is.EqualTo(expectedItems).UsingItemModelComparer());
+        }
+
+        [Test]
+        public async Task GetItem_ExistingId_ItemReturned()
+        {
+            var expectedItem = new ItemModel {Id = Guid.Parse("c5cc89a0-ab8d-4328-9000-3da679ec02d3"), Text = "Make a cofee"};
+
+            var result = await _controller.GetItem(Guid.Parse("c5cc89a0-ab8d-4328-9000-3da679ec02d3"));
+            var message = await result.ExecuteAsync(CancellationToken.None);
+            var actualStatusCode = message.StatusCode;
+            message.TryGetContentValue(out ItemModel actualItem);
```

Comment:
This 4 lines of code repeat quite often in tests; consider introducing some generic private method that would allow you to make this a one-liner (you can, for example, leverage C# 7's deconstruction and tuples syntactic sugar similarly to demonstrated [here](https://blogs.msdn.microsoft.com/seteplia/2017/11/01/dissecting-the-tuples-in-c-7/).